docs/server_config.md

```diff
@@ -0,0 +1,24 @@
+CodeChecker server configuration
+====================================
+
+The server's configuration is stored in the server's *workspace* folder, in
+`server_config.json`. This file is created, at the first start of the server,
+using the package's installed `config/server_config.json` as a template.
+
+> **NOTICE!** `session_config.json` file has been deprecated.
+
+Table of Contents
+=================
+* [Run limitation](#run-limitations)
+* [Authentication](#authentication)
+
+## Run limitation
+The `max_run_count` section of the config file controls the maximum storable
```

Comment:
"Maximum storable" can be misinterpreted with the meaning of "concurrent storage operations running at once".